### PR TITLE
Created methods GET, POST and DELETE

### PR DESCRIPTION
**Pull request by Rafal-wq**
In this PR I created first methods for adding, getting and deleting customers. There is change documentation in README.md file.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const { brandRouter } = require('./routers/item-brand');
+const { workerRouter } = require('./routers/workers');
+
+app.use(express.json());
+app.use(urlencoded({
+    extended: true,
+}));
+app.use(bodyParser.json({
+    type: 'application/json',
+}));
```
Comment (ilyaKarachun) [inline index.js]: express.json() and bodyParser.json you need for parse JSON incoming data? Then why u add both? U can leave only express.json

File: package.json [hunk 2]
```diff
@@ -4,9 +4,16 @@
   "description": "## Shop with clothes schema",
```
Comment (ilyaKarachun) [inline package.json]: It's application or schema?) Too much for schema I think))

File: registry/workers-registry.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+const { v4 : uuid } = require('uuid');
+const { pool } = require('../utils/db');
+
```
Comment (ilyaKarachun) [inline registry/workers-registry.js:3]: For me, better was divide BLL and operations procedure with db

File: routers/category.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    .patch('/categories/:category_id', async (req, res) => {
+
+        const category = await CategoryRegistry.categoryGetOne(req.params.category_id);
+
+        if (category === null) {
+            throw new Error('No such category');
+        }
+
+        await category.update;
+        res.status(200).json(category);
```
Comment (ilyaKarachun) [inline routers/category.js:41]: Maybe be better to add a field with a message to all responses

File: registry/workers-registry.js (hunk 3 above)
Comment (Rafal-wq) [reply registry/workers-registry.js:3]: I'm not pretty sure what is about, could you give me some more informations? :)

File: routers/category.js (hunk 4 above)
Comment (Rafal-wq) [reply routers/category.js:44]: You mean to create global constant with message use in all methods?

File: package.json (hunk 2 above)
Comment (Rafal-wq) [reply package.json]: Done

File: index.js (hunk 1 above)
Comment (Rafal-wq) [reply index.js]: Done

File: registry/category-registry.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            category_name: this.category_name,
+        });
+        return this.category_id;
+    }
+    static async listAll(){
+        const [ category ] = await pool.execute("SELECT * FROM `category` ORDER BY `category_name` ASC");
+        return category.map(obj => new CategoryRegistry(obj));
+    }
+    static async categoryGetOne(category_id){
+        const [ results ] = await pool.execute("SELECT * FROM `category` WHERE `category_id` = :category_id", {
```
Comment (ilyaKarachun) [inline registry/category-registry.js]: Bad practice select all

File: registry/workers-registry.js (hunk 3 above)
Comment (Rafal-wq) [reply registry/workers-registry.js:3]: Done

**Pull request merged**